src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -63,7 +63,7 @@ public void SerializeUnsigned(BinaryWriter writer)
 
         private JObject GetInvokeResult(byte[] script, Signers signers = null)
         {
-            using ApplicationEngine engine = ApplicationEngine.Run(script, signers, gas: settings.MaxGasInvoke);
+            using ApplicationEngine engine = ApplicationEngine.Run(script, container: signers, gas: settings.MaxGasInvoke);
```

Comment:
```cs
public static ApplicationEngine Run(byte[] script, StoreView snapshot = null, IVerifiable container = null, Block persistingBlock = null, int offset = 0, long gas = TestModeGas)
```
https://github.com/neo-project/neo/blob/master/src/neo/SmartContract/ApplicationEngine.cs#L290

`signers` will be assigned to `snapshot`.